docs/content/docs/deployment/resource-providers/native_kubernetes.md

```diff
@@ -584,4 +584,18 @@ When deploying Flink natively on Kubernetes, the following jars will be recogniz
 
 Please refer to the [Debugging Classloading Docs]({{< ref "docs/ops/debugging/debugging_classloading" >}}#overview-of-classloading-in-flink) for details.
 
+## Flink Kubernetes Operator
```

Comment:
I think instead of adding this to the bottom we could add one paragraph to the end of the introduction instead with a general link to the docs.